src/main/scala/io/iohk/ethereum/network/p2p/messages/PV64.scala

```diff
@@ -0,0 +1,132 @@
+package io.iohk.ethereum.network.p2p.messages
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.BlockHeaderImplicits._
+import io.iohk.ethereum.domain._
+import io.iohk.ethereum.network.p2p.{Message, MessageSerializableImplicit}
+import io.iohk.ethereum.rlp.RLPImplicitConversions._
+import io.iohk.ethereum.rlp.RLPImplicits._
+import io.iohk.ethereum.rlp._
+import org.bouncycastle.util.encoders.Hex
+
+/**
+  * This is temporal PV64 version, the real one will be implemented by ETCM-355
+  * This one will be probably PV66 in the future
+  */
+object PV64 {
+  object Status {
+    implicit class StatusEnc(val underlyingMsg: Status)
+        extends MessageSerializableImplicit[Status](underlyingMsg)
+        with RLPSerializable {
+      override def code: Int = Codes.StatusCode
+
+      override def toRLPEncodable: RLPEncodeable = {
+        import msg._
+        RLPList(
+          protocolVersion,
+          networkId,
+          chainWeight.totalDifficulty,
+          bestHash,
+          genesisHash,
+          chainWeight.lastCheckpointNumber
```

Comment:
This is really minor, but I see you keep insisting on having this as last field in the RLP. I really don't see the point in this. This is not an extension, all the fields could be totally different. It would make much more sense to me to keep relevant fields close together.